proto/src/determined/api/v1/api.proto

```diff
@@ -678,7 +679,9 @@ service Determined {
   rpc GetJobQueueStats(GetJobQueueStatsRequest)
       returns (GetJobQueueStatsResponse) {
     option (google.api.http) = {
-      get: "/api/v1/jobs/stats/{resource_pools}"
+      // FIXME could be queue-stats
+      // TODO how do we get stats for multiple resource pools? we have view that needs it
```

Comment:
@stoksc suggestions? ideally, we'd like to avoid setting up the clients to open on request per RP (UI design in figma in the ERD)

could do /resource-pools/stats but that'd conflict with {pool_id} unless we nest that under queue. 

sorta similar issue, but less important, for batch queue control actions across resourcepools

Reply:
I think `/api/v1/resource-pools/queues/stats?pool_id=default_cpu&pool_id=default_gpu` is a fine scheme (or comma separated `pool_ids=...` since some client take params as a map and dont properly handle multiples).

Reply:
that works we'd need to update the get part so it doesn't conflict with 
 `get: "/api/v1/resource-pools/{pool_id}/queue"`

> that'd conflict with {pool_id} unless we nest that under queue

maybe `/api/v1/resource-pools/queues/` with pool_id as query parameter?

Reply:
I did ponder `/api/v1/resource-pools/default,aux,gpu/queue/states` but idk, that's pretty non-standard.

Reply:
I updated the routes.